src/content/docs/data-apis/ingest-apis/metric-api/report-metrics-metric-api.mdx

```diff
@@ -269,7 +269,7 @@ Each metric data point map in the `metrics` array uses the following key-value s
       </td>
 
       <td>
-        <DNT>**Required**</DNT>. The metric's start time in [Unix time](https://currentmillis.com/). The default uses UTC time zone. This field also support seconds, microseconds, and nanoseconds. However, the data will be converted to milliseconds for storage and query. Metrics reported with a timestamp older than 48 hours ago or newer than 24 hours from the time they are reported are dropped.
+        <DNT>**Required**</DNT>. The metric's start time in [Unix time](https://currentmillis.com/). The default uses UTC time zone. This field also support seconds, microseconds, and nanoseconds. However, the data will be converted to milliseconds for storage and query. Metrics reported with a timestamp older than 48 hours in the past or more than 24 hours in the future from the time they are reported are dropped.
```

Comment:
```suggestion
        <DNT>**Required**</DNT>. The metric's start time in [Unix time](https://currentmillis.com/). The default uses UTC time zone. This field also support seconds, microseconds, and nanoseconds. However, the data will be converted to milliseconds for storage and query. Metrics are dropped if they have a timestamp more than 48 hours in the past or more than 24 hours in the future from the time they are reported.
```

Reply:
@sujitnewrelic I know this revision has passive voice, but in this case, that seems appropriate.